app/markets/ActivityTable.tsx

```diff
@@ -20,6 +20,12 @@ import {
 } from '@/utils';
 import { Button, Icon, Tag, TagColorSchemeProp } from '@swapr/ui';
 import { Outcome } from '@/entities';
+import { DuneClient, LatestResultArgs, ParameterType } from '@duneanalytics/client-sdk';
+import Image from 'next/image';
+
+const DUNE_API_KEY = process.env.NEXT_PUBLIC_DUNE_API_KEY || '';
```

Comment:
We should probably add this to `constants`

Reply:
Will move it. But not sure about moving constants used only in 1 place to a different file and export it there.